src/test/java/com/facebook/openwifirrm/modules/operationelement/VHTOperationElementTest.java

```diff
@@ -0,0 +1,51 @@
+package com.facebook.openwifirrm.modules.operationelement;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import com.facebook.openwifirrm.ChannelWidth;
+
+/** Test the Very High Throughput (VHT) Operation Element */
```

Comment:
you don't need a comment on these test classes, it's self-explanatory from the name